components/ww3/cime_config/config_compsets.xml

```diff
@@ -24,6 +24,11 @@
     <lname>2000_DATM%CFSR_SLND_SICE_SOCN_SROF_SGLC_WW3%sp36x36</lname>
   </compset>
 
+  <compset>
+    <alias>DTESTM-WW3</alias>
+    <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI_DOCN%SOM_DROF%JRA-1p4-2018_SGLC_WW3%sp36x36_TEST</lname>
```

Comment:
I'd recommend using JRA v1.5 rather than v1.4 here, so that the datm and drof modes are `JRA-1p4-2018` -> `JRA-1p5`.

The docn mode and ww3 mode are different here than in the GMPAS-JRA-WW3 compset pointed to above. I don't know enough to know what the differences are, but I'll revise my suggestion on the alias name above to include some sort of identifier if the intended compset here has differences from the GMPAS-JRA-WW3 one about to go in.

Reply:
Just noting my comment on docn doesn't make any sense given the other compset is a G-case.